evalml/tests/component_tests/test_simple_imputer.py

```diff
@@ -117,7 +117,7 @@ def test_simple_imputer_all_bool_return_original(data_type, make_data_type):
 
 @pytest.mark.parametrize("data_type", ['pd', 'ww'])
 def test_simple_imputer_bool_dtype_object(data_type, make_data_type):
-    X = pd.DataFrame([True, np.nan, False, np.nan, True], dtype=object)
+    X = pd.DataFrame([True, np.nan, False, np.nan, True], dtype='boolean')
```

Comment:
This update (and many of the others made in the tests) was made because now we try to convert the columns back to the original types. Before, the original dtype was specified as object, so the column would have been stored as a categorical col and we would have expected X_expected_arr to return a categorical col. We need to explicitly set to boolean for the expected outcome to be boolean. 

I think these conversions make sense over previous behavior--user input is boolean, so output is boolean.

Reply:
Agreed! Glad we don't have to do this anymore.